frontend/src/css/app.sass

```diff
@@ -100,6 +100,42 @@ p, div
   max-width: 50%
```

Comment:
*arbitrary comment location*

Thinking out loud here @garyghayrat and @mds1, but what if—instead of a separate page and route for batch send—the `/send` page had something like two tabs "Single Send" (the current UI) and "Batch Send" (the new batch send UI) defaulting to "Single Send"? Curious what you guys think of this and how hard/easy it would be to implement.

Reply:
I think it makes complete sense. I don't think we even need an extra tab, just add an `add another send` button right below the usual `Send` button. And depending on the `sendData` length we can either call the Umbra and do usual send or call batchSend contract for batch send.

Reply:
Question above using visuals, pick one:

![image](https://user-images.githubusercontent.com/61768337/236031789-2af69a39-c1a6-4cf7-824f-3631795f1d7b.png)

Reply:
If we want tabs, we can do this easily using: https://quasar.dev/vue-components/tab-panels#qtabpanels-api

One nice side effect is that tabs (or more generally, just one page/route and same way to toggle) should make it easier to dedupe logic and code.

Regardless of one page vs. separate pages, I still think the UI should be:
- form style on both desktop/mobile for a single send
- table style on desktop / card style on mobile for batch send

Reply:
So the left one?

Reply:
Ah sorry, seems you left that comment with the pics as I was writing my comment, so I did not see it. Yea, IMO the left one is better, since there's no reason to change the existing code UI (it already works and people are familiar with it)

Reply:
The update is looking great @garyghayrat! This is really starting to come together.

A couple quick questions:

1. Is it possible to make the tabbed nature of the interface a little more explicit? As it is, it's not particularly clear—especially on desktop mode—that the "Batch Send" text (which is greyed out and floating on the right) is even something I can click. Here are a couple of examples of interfaces that I think make this a bit clearer. How hard would it be to make style changes in this direction?

![image](https://github.com/ScopeLift/umbra-protocol/assets/1479802/b4542edb-f1c2-4a0a-b2f7-4219d3d521cd)

<img width="755" alt="Screen Shot 2023-05-10 at 4 14 58 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/b536219a-8edd-4481-af73-329766b70ae2">

2. How difficult would it be to completely remove the tabs on a network where the functionality is not available yet? If it's difficult I think we can live with the approach you have here, but I think simply showing the current page as-is for flagged networks would be ideal.

Reply:
Looking good! A few pieces of feedback:

--------

1. For the mobile view, we should either (1) have a single row with amount on the left and token dropdown on the right, or (2) keep the look the same as the send form. Looks like currently we're in a middle ground where the fields aren't full width but are on their own line. Can shorten the amount field placeholder to just `Amount` so it fits better

<img width="267" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/3a797e9e-050a-4df5-a202-3158ea967fca">

--------

2. Let's add top and bottom margin here so the cards aren't touching. Also it seems they get a gold outline when you touch the card (prob hover is the actual thing), but we don't need that since you can't actually click the card (i.e. the card itself is not actionable, unlike the home page cards where I think that border is from)

<img width="317" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/1c05d0ee-ef85-4755-b022-6bba627b3c2f">

--------

3. From the above screenshots and the one below, you can see that when I enter an amount, the field's placeholder text moves to the top. But this doesn't happen with the recipient field. We should make these consistent

<img width="847" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/e7663aac-fb19-41f6-b216-180f77e367cb">

--------

4. The "delete" icon in the above screenshot is just the letter X. Let's use the same close icon we use elsewhere in the codebase. Search `q-icon` usage of the fontawesome `fa-times` logo for examples

--------

5. The tabs are configured to be full width and take up too much space IMO, making them a bit hard to see. Can we center them to both take up less space, so they're closer together? Might look better if we move the header to the top and call it "Send", keep that constant, and below it are tabs called "Single Send" and "Batch Send"

<img width="1104" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/b8612912-aefd-477e-af46-d1f1e148e414">

Reply:
1. Added delete icon instead of "X" https://github.com/ScopeLift/umbra-protocol/pull/491/commits/f91f44bf054b3828fdbfed4cbbcd7a08ccc87a30
2. Changed mobile match send view to match the Send page style and remove golden outline/highlight https://github.com/ScopeLift/umbra-protocol/pull/491/commits/bd3bcdc635be649ca2742ea788fb9d7674d02f22
3. Removed tabs completely when batch send isn't supported https://github.com/ScopeLift/umbra-protocol/pull/491/commits/01c779b7811e9f712c798cbac20878d8822f5c5a
4. Used smaller numbers for desktop view https://github.com/ScopeLift/umbra-protocol/pull/491/commits/ff6469554e0f7f156e26e745347d66d5de7a61d3
5. Moved tabs under title and brought them closer to each other https://github.com/ScopeLift/umbra-protocol/pull/491/commits/749a4c043f2a9b778f7153a8c1f8774e54dd6b6a
6. Made input filed styles uniform on desktop https://github.com/ScopeLift/umbra-protocol/pull/491/commits/bac053ac2343b5984d1b2a7e1af316d7e118ccba
7. Fixed summary erroring out when sending 1k+ tokens https://github.com/ScopeLift/umbra-protocol/pull/491/commits/16d401071503c3b1fd70d820b2f16f0cb2a2bea8
8. Fixed flaky feature flag https://github.com/ScopeLift/umbra-protocol/pull/491/commits/b73e29003e80fc37162fd5193c445417561b70bc

Reply:
Great work @garyghayrat, on my first pass through it seems all these issues are resolved. I think the tabs look fine with the spacing fixed. I'll let @mds1 give everything a look to see if he agrees. This is really looking great!

Reply:
Yea tabs def look better now, thanks! From my numbering:
1. Resolved
2. Seems the gold outline is still there when I click using mobile view in chrome desktop, maybe that's not true on mobile? Connecting a wallet on localhost on mobile didn't work last time so we can just leave this for now and if it's there in prod it's not a big deal / not a blocker
3. Resolved
4. IMO they're still pretty big and take focus, but they shouldn't be a focus. They should be roughly the same size as the X, they don't need to be header elements. You can also try making them a shade of gray for less emphasis. Not gonna block on this though
5. Resolved

New one: there is a container/background of a different color on dark mode, but not on light mode, which we should unify. Prob no container for both?

<img width="957" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/3959d0a3-307f-4743-be24-5567dd13e686">

<img width="1013" alt="image" src="https://github.com/ScopeLift/umbra-protocol/assets/17163988/7cf11d4d-54c8-4939-bba8-d4d7fe73ec49">

Reply:
2. Gold outline should really be gone now https://github.com/ScopeLift/umbra-protocol/pull/491/commits/db8e00453c51adcc585c24922ddc9d5ee9816c40
4. Made 'X'es stand out much less https://github.com/ScopeLift/umbra-protocol/pull/491/commits/fbbb59333bb200d9ccf41551124f83ca721b9bc6
New one: good catch and not sure how it propped up but is fixed in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/f4570718db6b52e9fc529f84e66d0e689051ba00